### PR TITLE
fix bug with config loading

### PR DESCRIPTION
**Pull request by rocktavious**
## Issues
Fixes a bug from the viper upgrade but the error was hidden because of our bad error handling

## Changelog

- [X] List your changes here
- [X] Make a `changie` entry

### CODE REVIEW

File: src/cmd/root.go [hunk 1]
```diff
@@ -88,7 +88,10 @@ func readConfig() {
 	}
 	viper.SetEnvPrefix("OPSLEVEL")
 	viper.AutomaticEnv()
-	viper.ReadInConfig()
+	err := viper.ReadInConfig()
+	if err != nil {
+		panic(err)
```
Comment (rocktavious) [inline src/cmd/root.go:93]: This area of the code is called during cobra initialization so if this happens its OK to panic here

File: src/common/config.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ type Config struct {
 }
 
 type ConfigVersion struct {
-	Version string
+	Version string `json:"version" yaml:"version"`
```
Comment (rocktavious) [inline src/common/config.go:29]: not strictly needed but good to ensure the marshalar's work correctly

**Review by davidbloss — APPROVED** [review]
LGTM 🎉

**Review by taimoor-at-opslevel — APPROVED** [review]
`service preview` and `config view` WOMM

**Pull request merged**